packages/gatsby-theme-docs/src/templates/releases.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { MDXProvider } from '@mdx-js/react';
+import {
+  Markdown,
+  Subtitle,
+  ContentNotifications,
+} from '@commercetools-docs/ui-kit';
+import LayoutContent from '../layouts/content';
+import {
+  SEO,
+  Link,
+  ThemeProvider,
+  CodeExample,
+  MultiCodeExample,
+} from '../components';
+import PlaceholderMarkdownComponents from '../overrides/markdown-components';
+
+// See https://mdxjs.com/getting-started#table-of-components
+const components = {
+  // UI components
+  p: Markdown.Paragraph,
+  // NOTE: we want to ensure that only one h1 exists on each page.
+  // Therefore, we map the markdown header elements starting from h2.
+  // The h1 header will be automatically rendered based on the page title.
+  h1: Markdown.withAnchorLink(Markdown.H2),
+  h2: Markdown.withAnchorLink(Markdown.H3),
+  h3: Markdown.withAnchorLink(Markdown.H4),
+  h4: Markdown.withAnchorLink(Markdown.H5),
+  h5: Markdown.withAnchorLink(Markdown.H6),
+  h6: Markdown.withAnchorLink(Markdown.H6),
```

Comment:
Same here, in my opinion a release note should not have a heading structure in itself (Assuming this template is for the release note details page)

We can consider allowing one level to give a longer RN some structure (that would be H1 in MDX and sven would have to decide how to render it).  The rest should IMHO just render as paragraphs.